packages/flutter_tools/templates/create/pubspec.yaml.tmpl

```diff
@@ -1,6 +1,14 @@
 name: {{projectName}}
 description: {{description}}
 
+# The following defines the version and build number for your application.
+# A version number is three numbers separated by dots, like 1.2.43
+# followed by an optional build number separated by a +.
+# Both, version and build number, can be overwritten with flutter build command.
```

Comment:
"""
Both version and builder number may be overridden in `flutter build` by
specifying `--build-name` and `--build-number`, respectively.
"""